src/currency.h

```diff
@@ -70,11 +70,11 @@ enum Currencies {
 
 /** Specification of a currency. */
 struct CurrencySpec {
-	uint16 rate;
-	char separator[8];
-	Year to_euro;      ///< %Year of switching to the Euro. May also be #CF_NOEURO or #CF_ISEURO.
-	char prefix[16];
-	char suffix[16];
+	uint16      rate;      ///< The conversion rate compared to the base currency.
+	std::string separator; ///< The thousands separator for this currency.
+	Year        to_euro;   ///< %Year of switching to the Euro. May also be #CF_NOEURO or #CF_ISEURO.
```

Comment:
Please don't do this .. it will make future additions horrible to realign: not one, but two places to take care of :P

(talking about the alignment of variable type / variable name)